test/conformance/conformance_features.go

```diff
@@ -0,0 +1,37 @@
+// Copyright Envoy Gateway Authors
+// SPDX-License-Identifier: Apache-2.0
+// The full text of the Apache license is available in the LICENSE file at
+// the root of the repo.
+
+package conformance
+
+import (
+	"k8s.io/apimachinery/pkg/util/sets"
+	"sigs.k8s.io/gateway-api/conformance/tests"
+	"sigs.k8s.io/gateway-api/conformance/utils/suite"
+	"sigs.k8s.io/gateway-api/pkg/features"
+)
+
+// SkipTests is a list of tests that are skipped in the conformance suite.
+var SkipTests = []suite.ConformanceTest{
+	tests.GatewayStaticAddresses,
+	tests.GatewayHTTPListenerIsolation,          // https://github.com/kubernetes-sigs/gateway-api/issues/3049
+	tests.HTTPRouteBackendRequestHeaderModifier, // https://github.com/envoyproxy/gateway/issues/3338
+}
```

Comment:
we can move this under `internal` to avoid import test package.

Reply:
I tried to move `conformance_features.go` to `internal/gatewayapi` package, but that leads to an import cycle
```
package github.com/envoyproxy/gateway/internal/gatewayapi/status
	imports github.com/envoyproxy/gateway/internal/gatewayapi
	imports github.com/envoyproxy/gateway/internal/gatewayapi/status: import cycle not allowed
```

@zirain do you maybe have a suggestion regarding where you would like this file to go?
Do you want it straight under `internal` (`internal/conformance_features.go`)?

Reply:
what about `internal/gatewayapi/conformance/features.go`

Reply:
done